README.md

```diff
@@ -1,12 +1,46 @@
 # Self-healing Apache Zookeeper ECS cluster
 
 ## General information
-This repo contains terraform code and docker file which will allow you to create self-healing Apache Zookeeper cluster into AWS ECS
+The repository is a fork from https://github.com/dethmix/self-healing-zookeeper which the goal is to have a Zookeeper server that is self-healing and is deployed in AWS ECS, this has been further improved by removing the ECS autodiscovery feature as it creates complexity during deployment and automate the Zookeeper node enrollment.
 
 ## Usage
 Terraform code can be found in `terraform` directory, just update variable.tf file with values related to your AWS VPC and execute `terraform plan && terraform apply`
- 
-Dockerfile for building custom Apache Zookeeper image can be found in `docker` directory. It contains only few custom changes in comparing to the original Apache Zookeeper dockerfile. The main one is generation of `myid` from ECS service name.
 
-## Blog post
-More details about this setup can be found at https://tech.smartling.com/self-healing-apache-zookeeper-cluster-470b248ccb12
+
+## Infrastructure
+
+The infrastructure required to implement this solution is an EC2 container instance with a minimum of 3 subnets in which each subnet is equivalent to one container. 
+
+We also automated the Zookeeper node deployment by just adding a DNS instead of an IP addresses as the endpoint, this make sure that we are not changing the IP address again whenever we want to reroll the containers.
```

Comment:
We automated the Zookeeper node deployment by adding DNS  names in the configs instead of IP addresses while defining the members of the zookeeper cluster. This ensures that we don't have to change the IP address again when we want to deploy zookeeper.